<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMainWindow</class>
 <widget class="QMainWindow" name="GameMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <widget class="QGraphicsView" name="gameView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="leftArrowBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>200</number>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="downArrowBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>v</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>200</number>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="rightArrowBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>200</number>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="upArrowBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>200</number>
    </property>
    <property name="arrowType">
     <enum>Qt::UpArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="collectTreasure_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Collect</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="statistic_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
